Update for cisagov move

hillaryj opened the pull request:
Update URLs for move to `cisagov` org and remove unnecessary oauth url

## 🗣 Description

- Updates target URL for the move to the `cisagov` org
- Removes oauth from URL query to retrieve tarball since it's now a public repo

## 💭 Motivation and Context

Associated with https://github.com/cisagov/ansible-role-cyhy-core/issues/10 - removes deprecated oauth usage both due to deprecation as well as now the target is publicly accessible without an auth token.

## 🧪 Testing

<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## 📷 Screenshots (if appropriate)

## 🚥 Types of Changes

<!--- What types of changes does your code introduce? -->
<!--- Put an `x` in all the boxes that apply: -->

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

<!--- Go over all the following points, and put an `x` in all the
boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask.
We're here to help! -->

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

hillaryj committed:
Update for cisagov move

Update URLs for move to cisagov org and remove unnecessary oauth url

dav3r reviewed (changes requested):
What you have so far is good, but you also need to remove the Ansible variable for the OAuth token [here](https://github.com/cisagov/ansible-role-cyhy-core/blob/develop/vars/main.yml#L4-L5).

And since this repo uses our more modern [`molecule-iam-user-tf-module`](https://github.com/cisagov/molecule-iam-user-tf-module) and there is another SSM parameter still present, you can also easily clean up the terraform that creates the test user by removing `"/github/oauth_token", ` from [`iam_user.tf`](https://github.com/cisagov/ansible-role-cyhy-core/blob/develop/terraform/iam_user.tf#L13).  

I don't remember what happens if we call [`molecule-iam-user-tf-module`](https://github.com/cisagov/molecule-iam-user-tf-module) with an empty list of `ssm_parameters`, but I think it does not work as expected.

jsf9k committed:
Correct company name in meta/main.yml

jsf9k committed:
Rename vars directory to defaults

jsf9k committed:
Remove now-unused oauth token from Ansible variables

jsf9k committed:
Remove test user's access to now-unneeded oauth_token var

jsf9k committed:
Remove unnecessary prepare playbook

Boto3 is used to access SSM Parameter Store variables on the host
system, not the system to which the role is being applied.  Installing
it on the target system has no effect.

jsf9k commented:
> What you have so far is good, but you also need to remove the Ansible variable for the OAuth token [here](https://github.com/cisagov/ansible-role-cyhy-core/blob/develop/vars/main.yml#L4-L5).
> 
> And since this repo uses our more modern [`molecule-iam-user-tf-module`](https://github.com/cisagov/molecule-iam-user-tf-module) and there is another SSM parameter still present, you can also easily clean up the terraform that creates the test user by removing `"/github/oauth_token", ` from [`iam_user.tf`](https://github.com/cisagov/ansible-role-cyhy-core/blob/develop/terraform/iam_user.tf#L13).

I was anxious to perform a test build of cisagov/ansible-role-cyhy-commander#8 with these cyhy-core changes in place, so I went ahead and created several commits to address the issues you raised.

dav3r reviewed (approved):
Looks great now! 🚀

jsf9k reviewed (approved):
These changes look good to me!
